components/button/README.md

```diff
@@ -19,6 +19,7 @@ The `d2l-button` element can be used just like the native button element, but al
 |--|--|--|
 | `description` | String | A description to be added to the `button` for accessibility |
 | `disabled` | Boolean | Disables the button |
+| `disabledTooltip` | String | Tooltip text when disabled |
```

Comment:
If anyone has preference, we could alternatively call this `disabledText`.  Jeff and I talked briefly about both options but leaned towards `disabledTooltip` since it makes it clearer what it's for, even though it does tie it to tooltip.

Reply:
I'm OK with this. Normally I don't love tying components together like this, but "tooltip" is also a semantic thing so I'm good with it. If we were to add normal tooltips to buttons (which we wanted to do), I think we'd call it `tooltip` so that tracks.